### DISCUSSION TITLE
v2.4.0

### DISCUSSION THREAD

**Discussion by Zenda-Cross** [vega-org/vega-app#96] [Announcements]
- New Providers MultiStream and HiAnime
- Added Themes
- External Subs support
- Updated UI & UX
- Bugs fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/Zenda-Cross/vega-app/releases/tag/v2.4.0'>v2.4.0</a>.</em>

**Comment by mks32323232** [comment]
autoembed did not available.?

**Reply by Zenda-Cross** [reply]
> autoembed did not available.?

Its in MultiStream as a server

**Comment by adilurrhmn** [comment]
Autoembed was working fine.
But now multistream not working for hindi dubbed contents. Its showing only loading screen.
![Screenshot_2024-08-27-22-26-32-21_11fe5ff35dd1fae673697ec5eb78287f](https://github.com/user-attachments/assets/ca094567-2c21-4c41-969b-3c210477f56c)
![Screenshot_2024-08-27-22-26-38-68_11fe5ff35dd1fae673697ec5eb78287f](https://github.com/user-attachments/assets/233e063b-cff0-4a1f-be8a-ee49d786426f)

**Reply by himanshu8443** [reply]
I haven't changed anything in autoEmbed just moved to MultiStream
I have also checked and its working, maybe its server domain is blocked by your ISP

**Comment by LEOATIQ** [comment]
Can u add yts 1337x or kickasstorrent

**Reply by himanshu8443** [reply]
torrent can't be played directly it will require additional packages
so maybe in future

**Comment by mks32323232** [comment]
some server play video after 
cast  tv not working....

**Reply by Zenda-Cross** [reply]
> some server play video after 
> cast  tv not working....

can you explain it more ?

**Reply by mks32323232** [reply]
> > some server play video after 
> > cast  tv not working....
> 
> can you explain it more ?

streaming to tv Chromecast  some server 
not working ...

**Comment by LEOATIQ** [comment]
So many issue is there with 4k movie first it doesn't load and when u try to change to audio English 7.1 true HD it stops

**Reply by Zenda-Cross** [reply]
can you send a movie link for example.
Audio format must be unsupported by the player so use external player.

**Reply by LEOATIQ** [reply]
Kingdom of the planet of the apes

**Comment by LEOATIQ** [comment]
And in tv os when u pause the movie for some seconds it will be stuttering

**Reply by Zenda-Cross** [reply]
I regret to inform you that I won't be updating the TV app any longer.
